docs/src/problem.md

```diff
@@ -11,6 +11,60 @@
 Problem
 ```
 
+### Input/Output Sets
+
+Different solvers require problems formulated with particular input and output sets.
+The table below lists all of the solvers with their required input/output sets.
+
+ - HR = `Hyperrectangle`
+ - HS = `HalfSpace`
+ - HP = `HPolytope`
+ - PC = `PolytopeComplement`
+
+[](TODO: review these. Not sure they're all correct.)
+
+|        Solver        |  Input set  |    Output set    |
+|----------------------|:-----------:|:----------------:|
+| [`ExactReach`](@ref) | HP          | HP (bounded[^1]) |
+| [`AI2`](@ref)        | HP          | HP (bounded[^1]) |
+| [`MaxSens`](@ref)    | HP          | HP (bounded[^1]) |
```

Comment:
I suppose MaxSens takes both HP and HR. Or are we using HP as a more general object that includes HR?

Reply:
This is a distinction that I was struggling with as well. I put a note about it after the table, but I don't consider the point resolved... We may end up going with HP/HR/HS for each one that takes multiple inputs. The additional issue with solvers like MaxSens is that actually, for HP input, it _overapproximates_ as HR anyway, so saying it takes HP is only true with a caveat.